eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -53,6 +53,7 @@ steps:
           -GitUrl "https://$(azuresdk-github-pat)@github.com/${{ parameters.PROwner }}/${{ repo }}.git"
           -PushArgs "${{ parameters.PushArgs }}"
           -SkipCommit $${{parameters.SkipCheckingForChanges}}
+          -RemoteName 'azure-sdk-fork'
```

Comment:
We shouldn't need to pass this through as the script already defaults to this value.